python/insights_ansible_playbook_verifier/app.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import logging
+import pathlib
+import pkgutil
+import sys
+
+import insights_ansible_playbook_lib as lib
+
+logger = logging.getLogger(__name__)
+
+
+def read_playbook_from_stdin() -> str:
+    """Read playbook with to-be-verified plays from standard input."""
+    try:
+        result: str = "".join(line for line in sys.stdin)
+    except KeyboardInterrupt:
+        logger.error("Received keyboard interrupt.")
+        raise RuntimeError("Playbook loading was aborted.") from None
+    return result
+
+
+def read_revocation_playbook_from_package() -> str:
+    """Read revocation playbook content saved in the package."""
+    data: str = pkgutil.get_data(
+        "insights_ansible_playbook_verifier",
+        "data/revoked_playbooks.yml",
+    ).decode("utf-8")  # type: ignore
+    return data
+
+
+def get_gpg_key_content() -> bytes:
+    """Read the public GPG key to verify the plays with."""
+    data: bytes = pkgutil.get_data(
+        "insights_ansible_playbook_verifier",
+        "data/public.gpg",
+    )  # type: ignore
+    return data
+
+
+def run() -> None:
+    parser = argparse.ArgumentParser()
```

Comment:
since argparse is used now, i'd also make `--version` working; nothing super critical, a simple nice touch extra